test/integration/master/BUILD

```diff
@@ -8,7 +8,7 @@ load(
 
 go_test(
     name = "go_default_test",
-    size = "large",
+    timeout = "long",
```

Comment:
Actually, I think we should keep size="large" here. There are many test cases here, so more RAM is preferred.